### PR TITLE
Enforce max file count and active send button

### PR DESCRIPTION
**Pull request by csduarte**
#### Summary
1. Wait until all images are done uploading to enable the send button
2. Enforce max file upload count `MAX_IMAGE_COUNT = 5`

#### Checklist
- [x] Has UI changes
- [x] Includes text changes and localization file updates

## Behavior
max_files

## iOS
ios_max_files

## Android
android_max_files

@miguelespinoza

### CODE REVIEW

File: app/components/post_textbox/post_textbox.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ import Typing from './components/typing';
 const INITIAL_HEIGHT = Platform.OS === 'ios' ? 34 : 36;
 const MAX_CONTENT_HEIGHT = 100;
 const MAX_MESSAGE_LENGTH = 4000;
+const MAX_IMAGE_COUNT = 5;
```
Comment (enahum) [inline app/components/post_textbox/post_textbox.js]: can we make this constant `MAX_FILE_COUNT` ?

**Comment by miguelespinoza** [comment]
@enahum and @hmhealey does everything look good for this to get merged?

**Comment by enahum** [comment]
@miguelespinoza this one needs a rebase

**Comment by miguelespinoza** [comment]
@enahum done 👍

File: app/components/file_upload_preview/file_upload_preview.js [hunk 2]
```diff
@@ -186,5 +203,10 @@ const style = StyleSheet.create({
     scrollViewContent: {
         alignItems: 'flex-end',
         marginLeft: 14
+    },
+    warning: {
+        color: 'white',
```
Comment (enahum) [inline app/components/file_upload_preview/file_upload_preview.js]: Can we make this to use the error label theme color?

Comment (miguelespinoza) [reply app/components/file_upload_preview/file_upload_preview.js]: @enahum, I followed the error label theme color and it doesn't go well with the dark overlay when selecting images.  I passed this by @asaadmahmood and he gave the ok. We can bring him in this discussion to take a second look

**Comment by asaadmahmood** [comment]
@miguelespinoza Can we get a screenshot of  this with a dark theme (with the center channel dark).

**Comment by enahum** [comment]
@miguelespinoza do you have the screenshots requested by @asaadmahmood available?

**Comment by miguelespinoza** [comment]
@enahum, I'll post a screenshot today

**Comment by miguelespinoza** [comment]
@enahum && @asaadmahmood 

simulator screen shot - iphone 6 - 2017-12-14 at 12 32 04

**Comment by asaadmahmood** [comment]
Looks good to me 👍

**Comment by miguelespinoza** [comment]
Awesome, thanks @asaadmahmood.

@enahum, this should be good to go 😄

**Pull request merged**